_projects/implement_aspects.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - coala
-desc: "This project is about annotating results of bears with ``aspects`` which are like categories allowing to group results."
+desc: "This project is about annotating results of bears with ``aspects`` which are like categories allowing to group 
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.